examples/screenshot/src/main.rs

```diff
@@ -0,0 +1,81 @@
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")] // hide console window on Windows in release
+
+use eframe::{
+    egui::{self, ColorImage},
+    glow::{self, HasContext},
+};
+use itertools::Itertools;
+
+fn main() {
+    let options = eframe::NativeOptions::default();
+    eframe::run_native(
+        "Take screenshots and display with eframe/egui",
+        options,
+        Box::new(|_cc| Box::new(MyApp::default())),
+    );
+}
+
+#[derive(Default)]
+struct MyApp {
+    texture: Option<egui::TextureHandle>,
+    screenshot: Option<ColorImage>,
+}
+
+impl eframe::App for MyApp {
+    fn update(&mut self, ctx: &egui::Context, _frame: &mut eframe::Frame) {
+        egui::CentralPanel::default().show(ctx, |ui| {
+            if let Some(screenshot) = self.screenshot.take() {
+                self.texture = Some(ui.ctx().load_texture("screenshot", screenshot));
+            }
+
+            if ui
+                .add(egui::Label::new("Screenshot (hover me!):").sense(egui::Sense::hover()))
+                .hovered()
+            {
+                ctx.set_visuals(egui::Visuals::dark());
+            } else {
+                ctx.set_visuals(egui::Visuals::light());
+            };
+            if let Some(texture) = self.texture.as_ref() {
+                ui.image(texture, texture.size_vec2());
```

Comment:
nit: This is what it looks like on a high-dpi screen. I think it would be less confusing to instead show the screenshot scaled down to fit inside of `ui.available_size()`

<img width="912" alt="Screen Shot 2022-05-13 at 09 25 00" src="https://user-images.githubusercontent.com/1148717/168232733-3dd1cbb3-e7c8-4ea0-b0aa-89efab0b4c53.png">